CHANGELOG.md

```diff
@@ -5,6 +5,48 @@ information on the list of deprecated flags and APIs please have a look at
 https://docs.docker.com/engine/deprecated/ where target removal dates can also
 be found.
 
+## 1.12.2-rc1 (2016-09-27)
+
+**IMPORTANT**: Docker 1.12 ships with an updated systemd unit file for rpm
+based installs (which includes RHEL, Fedora, CentOS, and Oracle Linux 7). When
+upgrading from an older version of docker, the upgrade process may not
+automatically install the updated version of the unit file, or fail to start
+the docker service if;
+
```

Comment:
Oh, there's more instructions below this (sorry, it's quite lengthy) https://github.com/docker/docker/blame/master/CHANGELOG.md#L10-L36